### PR TITLE
Feat: add login form to landing page

### LINKED ISSUE #3: Add login functionality to the website with input boxes
As a user, i want to see input boxes when i visit the sauce demo webpage

Image

### PR DESCRIPTION
**Pull request by testerremoteee43**
I have added new input boxes for login username and password.
Everything should look perfect and work

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,2 +1,21 @@
+<html>
+
+   <h1>I am sauce demo site, source: believe me</h1>
```
Comment (testerremoteee43) [inline index.html:3]: why should i believe you

File: index.html [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+        <form>
+            <div>
+                
+                <input placeholder="Username">
+            </div>
+            <div>
+                <input placeholder="Password">
+            </div>
+            <button>
+                Sign In
```
Comment (testerremoteee43) [inline index.html:15]: i think you should change this to login

File: index.html (hunk 1 above)
Comment (testerremoteee43) [reply index.html:3]: trust me bro

**Review by JP100593 — APPROVED** [review]

**Comment by karamulkin** [comment]
Look s good

**Comment by cxrvd** [comment]
i wont believe it

**Pull request merged**